### DISCUSSION TITLE
Can someone help me downlaoding the sniper

### DISCUSSION THREAD

**Discussion by kurschlo** [NoMeansNowastaken/KarutaSniper#9] [Q&A] [answered]
Hey,
Can someone help me to download the Karuta Sniper. I have tried it by myself but it wont work. I have never done something like that. I would really appreciate it.

**Comment by splat58** [comment, marked as answer]
<img width="1436" alt="Screen Shot 2023-02-12 at 22 40 28" src="https://user-images.githubusercontent.com/91102438/218388777-c9d7de3d-f357-4344-aa2b-096e0322e2cd.png">
<img width="1429" alt="Screen Shot 2023-02-12 at 22 42 43" src="https://user-images.githubusercontent.com/91102438/218388963-09d479b2-bc9d-4698-a326-3ce64d52e332.png">

**Reply by kurschlo** [reply]
I mean downloading the requierments and how to link them.

**Reply by splat58** [reply]
do this in command prompt whilst in the karuta sniper directory

```pip install -r requirements.txt```

**Reply by kurschlo** [reply]
and how do I download the repo?

**Reply by splat58** [reply]
follow the images above

**Reply by kurschlo** [reply]
I downloaded a repo named archlinuxcn, do I need to do something with it?

**Reply by NoMeansNowastaken** [reply]
Not sure how you got there but if you are using linux just open a terminal and do ```git clone https://github.com/NoMeansNowastaken/KarutaSniper.git``` and then ```cd KarutaSniper```
